app/socket-apis/__tests__/get-points-of-ids.js

```diff
@@ -0,0 +1,139 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+import { MongoMemoryServer } from 'mongodb-memory-server'
+import { Mongo } from '@enciv/mongo-collections'
+import { MongoClient, ObjectId } from 'mongodb'
+import Points from '../../models/points'
+import getPointsOfIds from '../get-points-of-ids'
+
+let memoryServer
+let db
+
+const synuser = { synuser: { id: 'user1' } }
+
+beforeAll(async () => {
+  memoryServer = await MongoMemoryServer.create()
+  const uri = memoryServer.getUri()
+  await Mongo.connect(uri) // Connect to Mongo using the provided MongoMemoryServer URI
+  db = Mongo.db
+  Points.setCollectionProps() // Set collection props if not already done in the model
+})
+
+afterAll(async () => {
+  await Mongo.disconnect()
+  await memoryServer.stop()
+})
+
+beforeEach(async () => {
+  jest.spyOn(console, 'error').mockImplementation(() => {})
+})
+
+afterEach(async () => {
+  console.error.mockRestore()
+})
+
+// Test case 1: Nothing found
+test('Nothing found', async () => {
+  const callback = jest.fn()
+
+  // Pass synuser inside ids array
+  await getPointsOfIds([{ synuser: { id: 'user1' } }, new ObjectId(), new ObjectId()], callback)
```

Comment:
This must be `await getPointsOfIds/call(..`

Please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call